pkg/process/procutil/data_scrubber_fallback_test.go

```diff
@@ -0,0 +1,61 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !windows
+
+package procutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStripArguments(t *testing.T) {
+	for _, tc := range []struct {
+		name     string
+		cmdline  []string
+		expected string
+	}{
+		{
+			name:     "OS parse",
```

Comment:
Why are these called OS parse?

Reply:
These follow from case 1 of the [Jira ticket](https://datadoghq.atlassian.net/browse/PROCS-1451) where the OS has already done the splitting for us. I can rename it to something more descriptive if that is preferred!

Reply:
Ok, I understand this better now.